apps/web/src/components/Setting.tsx

```diff
@@ -105,7 +111,7 @@ function Setting<T = "switch" | "dropdown", K = unknown>({
       {/* Dropdowns */}
       {type === "dropdown" ? (
         <Dropdown
-          style={{ minWidth: 82 }}
+          style={{ minWidth: minWidth ?? 82 }}
```

Comment:
I added the `minWidth` property to the Setting because I wanted to maintain any backwards compatibility while also patching it so I could make the text in my setting box fit as before it would just end with `...` which did not look very good.